src/main/resources/config/config.yaml

```diff
@@ -33,6 +33,7 @@ database:
   properties:
     charSet: UTF-8
     hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect
+    hibernate.generate_statistics: false
```

Comment:
So this is where you’re supposed to put it?

(Previous attempt: https://github.com/alphagov/pay-webhooks/pull/373)